workflows/templates/wipe-and-reboot-worker.yaml

```diff
@@ -49,14 +83,15 @@ spec:
                 value: "{{inputs.parameters.dryRun}}"
               - name: scriptContent
                 value: |
-                  TOKEN=$(curl -k -s -S -d grant_type=client_credentials \
-                    -d client_id=admin-client \
-                    -d client_secret=`kubectl get secrets admin-client-auth -o jsonpath='{.data.client-secret}' | base64 -d` \
-                    https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token | jq -r '.access_token')
+                  curl -k -s -d grant_type=client_credentials \
+                      -d client_id=admin-client \
+                      -d client_secret="$(kubectl get secrets admin-client-auth -ojsonpath='{.data.client-secret}' | base64 -d)" \
+                      https://api-gw-service-nmn.local/keycloak/realms/shasta/protocol/openid-connect/token > /tmp/setup-token.json
+                  export CRAY_CREDENTIALS=/tmp/setup-token.json
```

Comment:
`CRAY_CREDENTIALS` env variable must be exported for `cray bss bootparameters list` below to work.